integration/deploy/compose_test.go

```diff
@@ -0,0 +1,308 @@
+//go:build integration
+// +build integration
+
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/okteto/okteto/integration"
+	"github.com/okteto/okteto/integration/commands"
+	"github.com/okteto/okteto/pkg/registry"
+	"github.com/stretchr/testify/require"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+const (
+	composeTemplate = `services:
+  app:
+    build: app
+    entrypoint: echo -n $RABBITMQ_PASS > var.html && python -m http.server 8080
+    ports:
+      - 8080
+      - 8913
+  nginx:
+    image: nginx
+    volumes:
+      - ./nginx/nginx.conf:/tmp/nginx.conf
+    entrypoint: /bin/bash -c "envsubst < /tmp/nginx.conf > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
+    environment:
+      - FLASK_SERVER_ADDR=app:8080
+    ports:
+      - 81:80
+    depends_on:
+      app:
+        condition: service_started
+    container_name: web-svc
+    healthcheck:
+      test: service nginx status || exit 1
+      interval: 45s
+      timeout: 5m
+      retries: 5
+      start_period: 30s`
+	stacksTemplate = `services:
```

Comment:
I would include a blank line here to separate 2 manifests. I think it reads better